src/views/product-downloads-view/components/downloads-section/index.tsx

```diff
@@ -208,17 +211,31 @@ const DownloadsSection = ({
   )
 
   return (
-    <article className={s.root}>
+    <div className={s.root}>
```

Comment:
Changed this to `div` because the other sections on the page don't use `article`. Rather than update all the sections to use `article` now, I'd prefer to double-check whether `section` or `article` should be used for these section components later and update them then.